docs/examples/gitHubApp.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+# this is the jenkins id.
+  name: "another-test-githubapp"
+  labels:
+# so we know what type it is.
+    "jenkins.io/credentials-type": "gitHubApp"
+  annotations:
+# description - can not be a label as spaces are not allowed
+    "jenkins.io/credentials-description" : "credentials from Kubernetes"
+type: Opaque
+stringData:
+  appID: 1234
+  # Base64 encoded private key
+  privateKey: LS0tLS1CRU...
```

Comment:
Does this s example work? PrivateKey would contain newlines would it not, so could never be not base64 encoded and so this would need to be in `data` not `stringData` iiuc

Reply:
Update the example to have an unencoded key